Test/Unit/String.php

```diff
@@ -195,7 +197,8 @@ public function case_collator()
     {
         $this
             ->given(
-                $this->function->setlocale = 'fr_FR',
+                $this->getPhpMocker()->setDefaultNamespace('Hoa\Ustring'),
+                $this->function->setlocale = OS_WIN ? 'fr' : 'fr_FR',
```

Comment:
On Windows, `fr_FR` seem to not be recognized(Windows value is `French_France`).
So `en_US` is used instead and break the test.
If someone can confirm that we should apply a similar patch to `Hoa\Ustring`.